ipython/ResamplingAndMonteCarloMethods/resampling_tutorial_2a.ipynb

```diff
@@ -27,9 +27,9 @@
    "id": "8525c260-6699-4f90-b20c-85e0d948bb3a",
    "metadata": {},
    "source": [
-    "In the paper, Wilcoxon describes a test to assess whether the two samples are drawn from the same population that is now commonly described as a *nonparametric* version of the independent sample t-test - that is, a version of the t-test that does not make the normality (or any particular distributional) assumption. \n",
+    "In the paper, Wilcoxon describes a test to assess whether the two samples are drawn from the same population that is now commonly described as a *nonparametric* version of the independent sample t-test - that is, a version of the t-test that does not make a normality (or any particular distributional) assumption. \n",
     "\n",
-    "Suppose we want to test that null hypothesis that the samples are drawn from the same distribution against the alternative that they are drawn from different distributions which tend to produce samples with a lower values of the statistic. Under certain assumptions, this can be argued as evidence that the location of the distribution underlying `x` is less than the location of the distribution underlying `y`. We pass the data into [`scipy.stats.mannwhitneyu`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.mannwhitneyu.html) with `alternative='less'`."
+    "Given samples `x` and `y`, Wilcoxon introduces a statistic which estimates the probability that a random observation from the distribution underlying `x` will be less than a random observation from the distribution underlying `y`. Suppose we want to test the null hypothesis that the samples are drawn from the same distribution against the alternative that they are drawn from different distributions which tend to produce samples that give lower values of the statistic. Under certain assumptions, this can be argued as evidence that the location of the distribution underlying `x` is less than the location of the distribution underlying `y`. To peform this test we pass the data into [`scipy.stats.mannwhitneyu`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.mannwhitneyu.html) with `alternative='less'`."
```

Comment:
I thought it was a little too terse to not mention what the statistic is at all, but it seems like it could break the flow to give the exact formula. As a compromise how about this intuitive sounding explanation?

Reply:
OK, but I don't think it's accurate to say it's a "statistic which estimates the probability..." The statistic itself is not an estimate of any probability. I would either describe how the statistic is calculated or state what it is "sensitive" to.

Reply:
What I meant is that in this case the statistic is literally the empirical estimate of this probability. One looks at the sampling distribution of the empirical estimate of this probability under the null assumption of exchangeability. There's probably a better way to phrase it. Like 


Wilcoxon introduces as a statistic an empirical estimate of the probability that a random observation from the distribution underlying `x` will be less than a random observation from the distribution underlying `y`.


Does that read better?

Reply:
Are we looking at the same paper/test?

![image](https://user-images.githubusercontent.com/6570539/219905790-f04abe0a-e02d-4131-9948-94c02ff0a8b1.png)

I'm talking about the rank sum statistic (magnitude is 24). I consider this to be the statistic of the paper, and the probability under the null assumption of exchangeability is the p-value. From a certain perspective, you can always call a p-value a "statistic", but I find it more meaningful to make a distinction between the statistic and the p-value associated with it.

(One interesting case of this distinction is Boschloo's test, in which the statistic is literally the p-value of Fisher's exact test. The statistic in Fisher's exact test, which also calculates a p-value under the null hypothesis of exchangeability, could be considered to be the sample odds ratio.)

Reply:
Ah, right. I should have double checked. If $U$ is the statistic then the empirical probability estimate  is
$$\hat{P} = \frac{U}{n_1*n_2}$$

where $n_1$ and $n_2$ are the respective group sizes.

Reply:
nevermind sorry, i'm thinking of the wilcoxon rank sum, this is wilcoxon signed rank

Reply:
wait, but you're doing the rank sum test in the tutorial, not the signed rank test from your image

Reply:
I picked the wrong screenshot above:
![image](https://user-images.githubusercontent.com/6570539/219908469-2c92ee8d-6f13-40fe-9602-966fbb5072be.png)

The statistic is an integer, though.

Reply:
Right, the empirical probability estimate is $\frac{U}{n_1n_2}$ where $n_1$ and $n_2$ are the respective group sizes. $U$ itself is an integer (it may not be in case of ties)

Reply:
I think of the intuitive idea behind the test being that we're looking at the sampling distribution of an empirical probability estimate under the null, but for calculation reasons, we only need to compute $U$.  In any case, what I wrote above is not right so I'll need to think a bit about what to say instead.

Reply:
I rewrote the phrase to say the statistic " is proportional to an empirical estimate of the probability that a random observation from...". Let me know what you think